.github/workflows/ok-to-test.yml

```diff
@@ -0,0 +1,36 @@
+# If someone with write access comments "/test-with-secrets" on a pull request, emit a repository_dispatch event
+name: Ok To Test
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  test-with-secrets:
+    runs-on: ubuntu-latest
+    # Only run for PRs, not issue comments
+    if: ${{ github.event.issue.pull_request }}
+    steps:
+    # Generate a GitHub App installation access token from an App ID and private key
+    # To create a new GitHub App:
+    #   https://developer.github.com/apps/building-github-apps/creating-a-github-app/
+    # See app.yml for an example app manifest
+    - name: Generate token
+      id: generate_token
+      # fork of tibdex/github-app-token
+      uses: nineinchnick/github-app-token@1364451f2a210ee26ab33b28f56f0e654bc63cd8
```

Comment:
@hashhar wait with the merge, we forgot to switch this action to the trinodb/github-actions repo

Reply:
Done